ophyd/device.py

```diff
@@ -1667,6 +1668,11 @@ def _repr_info(self):
         yield ("read_attrs", self.read_attrs)
         yield ("configuration_attrs", self.configuration_attrs)
 
+    # Placeholder required for devices to conform to the Bluesky Moveable protocol
+    # The actual implementation will be provided by sub classes
+    def set(self, *args, **kwargs) -> Any:
+        return super().set(*args, **kwargs)
```

Comment:
These placeholder `set()` functions that just forward on to `super()` don't seem to break any tests so they seem OK but might need some extra scrutiny because they change the public API I suppose

Reply:
Why is this needed? If `set` is just calling `super`'s set, and it hasn't refined the return type at all, then what is it for? If we leave this out then`super()`'s `set` would be used instead anyway.

Reply:
Good point. It made the type checker happy, but now that I look again, I don't think the base classes even have an implementation of `set`. `Component` doesn't even have a base class.

I'll have to have another look at this. Maybe adding `set()` to `BlueskyInterface` would make more sense.